inputs/BinaryOrbitAMR.in

```diff
@@ -3,7 +3,7 @@
 # *****************************************************************
 geometry.prob_lo     =  -2.5e13 -2.5e13 -2.5e13
 geometry.prob_hi     =   2.5e13  2.5e13  2.5e13
```

Comment:
Not sure if you meant, but looks like you have changed things here from periodic in x only to periodic in both x and y.